pwiz_tools/Skyline/TestRunner/Program.cs

```diff
@@ -656,7 +658,8 @@ private static void CheckDocker(CommandLineArgs commandLineArgs)
         private static string LaunchDockerWorker(int i, CommandLineArgs commandLineArgs, ref string workerNames, bool bigWorker, long workerBytes, int workerPort, StreamWriter log)
         {
             var pwizRoot = Path.GetDirectoryName(Path.GetDirectoryName(GetSkylineDirectory().FullName));
-            string workerName = bigWorker ? $"docker_big_worker_{i}" : $"docker_worker_{i}";
+            var invocationDetail = GetInvocationDetail(); // Avoids conflicts between this and any previous invocation

```

Comment:
GetInvocationDetail() seems like a pretty vague name. Why not `var timestamp = GetWorkerNameTimestamp()`" or something self-documenting like that? But this is nitpicking.

Reply:
I was going for not exposing implementation details, but your way is good too, I'll go ahead and call it something more descriptive.